r-archive-feedstock/recipe/meta.yaml

```diff
@@ -55,7 +54,6 @@ requirements:
     - r-base
     - {{native}}gcc-libs         # [win]
     - r-cli
-    - r-cpp11
```

Comment:
Is this because of [this commit](https://github.com/r-lib/archive/commit/1da20716f8114921144bffa178f6196d0dddbe2b)? do we want to unvendor r-cpp11 instead?

Reply:
There are no binaries included. They include only headers and added a flag for `PKG_CPPFLAGS = -Icpp11/include`. I don't think we should unvendor r-cpp11 here. I wonder if we ever patched R packages. The maintainers intentionally removed the dependency from the Description file.
Our `r-cpp11 0.4.6` on [linux-64](https://anaconda.org/r/r-cpp11) is behind the current version [0.5.0](https://cran.r-project.org/web/packages/cpp11/index.html) and it isn't easy to say what's behind the decision of vendoring r-cpp11 (the PR doesn't have any notes https://github.com/r-lib/archive/pull/103 and also the Release notes https://github.com/r-lib/archive/blob/main/NEWS.md#archive-118).

Reply:
ok, fine